tests/ClientBuilderTest.php

```diff
@@ -272,8 +223,8 @@ public function testClientBuilderSetsSdkIdentifierAndVersion(): void
     {
         $callbackCalled = false;
 
-        $clientBuilder = new ClientBuilder();
-        $clientBuilder->setBeforeSendCallback(function (Event $event) use (&$callbackCalled) {
+        $options = new Options();
```

Comment:
Instead of instantiating the `Options` object I would prefer to stick with the plain `array` of options